README.md

```diff
@@ -11,9 +11,7 @@ If you want to build Nexerelin locally, follow the steps below.
 Intellij and Eclipse forks are currently the most popular Java IDE out there.
 The following contains instructions on how to set this project up on Intellij. (Eclipse should be similar but you will need to figure out where those dialogs and settings are on your own.)
 
-First of all, Nexerelin also requires several other external libs for compiling. Check out the forum post for the list:
-[http://fractalsoftworks.com/forum/index.php?topic=9175.1800](http://fractalsoftworks.com/forum/index.php?topic=9175.1800)
-Download and unzip them somewhere on your computer, you can download them to the starsector mod folder but you don't have to load them in the launcher.
+First of all, Nexerelin also requires several other external libs for compiling. The whole list can be found at the [end of this document](#required-libraries). Download and unzip them somewhere on your computer, you can download them to the starsector mod folder but you don't have to load them in the launcher.
 This is just so the jar files contained in those mods can be loaded by nexerelin java classes.
 
 You will also need to use jdk1.7 to compile the jar; starsector is currently using jdk1.7.0.79 but anything newer than that would work as well. i.e. 1.7.0.80 and not java 8, 9, etc.
```

Comment:
Seemed to me that you also neeed a newer version of java to succesfully build, 7 fails due to the use of list.sort() in here

https://github.com/Histidine91/Nexerelin/blob/d75a285569a5a1a319824b49eec03a2f35b0a7a4/jars/sources/ExerelinCore/exerelin/campaign/SectorManager.java#L1006-L1010

Got it working with 11, but I dont know if theres a prefered version

Reply:
Yeah, that's a mistake on my part (should be `Collections.sort(collection, comparator)` to work in Java 7) which I've fixed in a currently unpushed commit.

Reply:
(now it's pushed, 1b5b6f24cb0ed0595180ae9f7d5f1476b71f241a)